src/relation_handlers.py

```diff
@@ -669,3 +669,45 @@ def update_broker_data(self, data, event):
         """For RadosGW, we want to change the key name and set the FSID."""
         data["fsid"] = self._get_fsid()
         data[self.key_name + "_key"] = data.pop("key")
+
+
+class CephMdsProviderHandler(CephClientProviderHandler):
+    """Handler for the ceph-mds relation."""
+
+    def __init__(self, charm, callback_f):
+        super().__init__(charm, "mds", callback_f)
+        self.mds_name = ""
+
+    @staticmethod
+    def _select_relation(relations, relation_id):
+        for relation in relations:
+            if relation.id == relation_id:
+                return relation
+
+    @staticmethod
+    def _remote_unit_name(client_name):
+        return "ceph-radosgw/" + client_name.split("-")[-1]
```

Comment:
wrong unit name (should be `ceph-fs`)

Reply:
Will remove. I think it's dead code anyways.